code/modules/integrated_electronics/~defines/~defines.dm

```diff
@@ -0,0 +1,37 @@
+#undef IC_INPUT
```

Comment:
what even is this file

Reply:
Likely a leftover of copy-pasting from polaris